### DISCUSSION TITLE
SingleLineText hidden options

### DISCUSSION THREAD

**Discussion by fewrainbows** [nocodb/nocodb#6808] [Q&A]
Hello!

what a software : )

Question:

In SingleLineText > Show more, I get this further options...

What do they mean?

![immagine](https://github.com/nocodb/nocodb/assets/88545208/4e04a224-6d3d-449b-97a3-4e1a1e1e6844)

I couldn't find any info on the help..

Thank you in advance!!

**Comment by dstala** [comment]
@fewrainbows Those are additional database specific configurations. We will add these to our documentation

NN - Not NULL
PK - Primary Key
AI - Auto Increment
UN - Unsigned
AU - Auto Update (Timestamp)

**Comment by fewrainbows** [comment]
Whoa!
thanks @dstala !

Please a small hint on "Type in Db and Lenght/value"

You see I'm not a pro 🙃
